components/dashboard/src/components/UsageView.tsx

```diff
@@ -310,7 +310,7 @@ function UsageView({ attributionId }: UsageViewProps) {
                                                     </div>
                                                     <div className="my-auto" />
                                                     <div className="flex flex-col col-span-3 my-auto">
-                                                        <span className="text-gray-400 dark:text-gray-500 truncate font-medium">
+                                                        <span className="text-gray-400 text-right dark:text-gray-500 truncate font-medium">
```

Comment:
did you also want to remove the truncate (elipses)?

Reply:
I was planning to do that later as I need to run for a meeting but if it's that simple, can do :)

Reply:
Done now

Reply:
the original issue complained about truncation on the time not the credits.
<img width="992" alt="Screenshot 2022-09-21 at 10 12 27" src="https://user-images.githubusercontent.com/849592/191452356-209e4a7f-bc26-4e28-bf1a-0d471e0bb2ce.png">

Reply:
Those are 2 different PRs, and 2 different Issues:
This one is https://github.com/gitpod-io/gitpod/issues/13154 which deals with alignment and truncation. There's another PR which deals with Credit rounding - https://github.com/gitpod-io/gitpod/issues/13056

Reply:
Yes, I'm talking about the truncation of the time property. See screenshot ☝️

Reply:
Yes, that should be fixed in https://github.com/gitpod-io/gitpod/pull/13173/files#diff-f54b0b6b3eff92d9ca581427987a0fc42b091a046375f29e76283ec615370c99R306

Reply:
you emoved the truncate for the 'usage' not the 'time'.

Reply:
I'm sorry, I don't follow what you mean. I'm removing truncation from the number of minutes that a workspace ran (as per the [issue](https://github.com/gitpod-io/gitpod/issues/13154)), not the usage record timestamp.

In the preview env, it renders as below, which should be the expected outcome
<img width="1047" alt="Screenshot 2022-09-21 at 17 13 04" src="https://user-images.githubusercontent.com/1419286/191542860-fdddda5a-fc68-4a4f-ba84-5e1cae6f425a.png">